tests/bogusObituaries.test.js

```diff
@@ -0,0 +1,27 @@
+const expect = require('chai').expect
+const { janusObituaryPage } = inject();
+
+Feature('bogusObituaries')
+    //tag for onboarding tasks
+    .tag('@onboarding')
+    //.tag('@currentTest')
+    .tag('@obitPage')
+    .tag('@REST')
+    .tag('@only-qa');
+
+//Data-Driven test
+let obituaries = new DataTable(['url', 'status']);
+
+obituaries.add(['https://www.qa-legacy.com/us/obituaries/chicagotribune/name/00', '404']);
+obituaries.add(['https://www.qa-legacy.com/us/obituaries/chicagotribune/name/01', '404']);
+
+//Pass dataTable to Data()
+Data(obituaries).Scenario('Verify that a user cannot navigate to a incorrect obit link', async ({ I }) => {
+
+    const data = await I.sendGetRequest('/us/obituaries/chicagotribune/name/00')
```

Comment:
You've hardcoded the url here. To make this test work, you'll need to use the `current` object to fetch the url/status from the datatable.